package.json

```diff
@@ -14,8 +14,8 @@
 	"bugs": "https://github.com/redhat-developer/vscode-openshift-tools/issues",
 	"engines": {
 		"vscode": "^1.64.2",
```

Comment:
The first version of VS Code that includes NodeJS 18 appears to be 1.82: https://code.visualstudio.com/updates/v1_82 . This version was released under a year ago. I don't know how old of a version of VS Code we are supposed to support.

If we are making this upgrade, we should probably list the engine version as `^1.82.3` .

Reply:
I think users are still able to use vscode 1.64 together with Node.js > 16.x, aren't they?

On the other hand, vscode 1.64 is out of support, so, I'm +1 for this change. 

Fixed

Reply:
My understanding is that VS Code uses whatever version of nodejs that the version of electron they depend on uses. Since fetch wasn't build into Node until Node 18, the first version of vscode that supports is whatever version added the version of Electron that uses Node 18. From what I can tell, that is `1.82.0`.

Node 16 is out of support, so I think it's safe to upgrade the engine to `1.82.0`, but we *should* change the engine version, since `fetch` isn't built in to Node 16.